src/interface/hooks/use-post-multitenancy-wallet.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * It creates a function that will post a schema to the server.
+ */
+import { useCallback, useState } from 'react'
+import { useAuth0 } from '@auth0/auth0-react'
+
+import post from '../api/helpers/post'
+
+import { WALLET_TYPE } from '../../utils/env'
+import WALLET_STATUS from '../../utils/wallet-status'
+
+const mkWalletKey = () => {
```

Comment:
what are the chances of getting same key?

Reply:
this is cryptographically secure so essentially zero. It's a 32 byte key so `1 in 115792089237316195423570985008687907853269984665640564039457584007913129639936`